INN/umbrella-energynewsnetwork
# Digest Search revisions, part 2

## Changes

This pull request makes the following changes:

- removes the horizontal-byline mixin as applied to archives and searches, so that we can add the byline date to the `/category/digest/` archive easily
- Adds the date to bylines on `/category/digest/` via a term-specific case in this theme's custom `largo_byline()` function
- adds a custom template partial used only on `category-digest.php` for results there, which disables the excerpt

![Screen Shot 2020-04-24 at 19 40 52](https://user-images.githubusercontent.com/1754187/80265470-ad90fa80-8665-11ea-9ca9-728aff06764c.png)


## Why



For #96

## Testing/Questions

Features that this PR affects:

- byline styles on all archive pages that aren't using the custom region archive template
- padding on the bottom of `.entry-content` on all archive pages that aren't using the custom region archive template
- bylines of posts displayed on `/category/digest/`


Questions that need to be answered before merging:

- [x] are styles broken on any archives?

Steps to test this PR:

1. verify that `/category/digest/` has dates in its bylines
2. check that other categories, like `/category/opinion/`, don't have dates or broken whitespace
